storage/range.go

```diff
@@ -940,6 +939,8 @@ func (r *Range) EndTransaction(batch engine.Engine, args *proto.EndTransactionRe
 			reply.SetGoError(r.splitTrigger(batch, args.SplitTrigger))
 		} else if args.MergeTrigger != nil {
 			reply.SetGoError(r.mergeTrigger(batch, args.MergeTrigger))
+		} else if args.ChangeReplicasTrigger != nil {
+			// ChangeReplicasTrigger is processed on input, not output.
```

Comment:
Yeah, I agree we need to call changeReplicasTrigger to update r.Desc. I specifically want to avoid accepting an updatedDesc in ChangeReplicas, however, because I don't want to have to compare the new and existing versions to make sure there is exactly one difference.